pkg/github/prcreation/prcreation.go

```diff
@@ -71,7 +71,7 @@ func (o *PRCreationOptions) UpsertPR(localSourceDir, org, repo, branch, prTitle,
 	stdout := bumper.HideSecretsWriter{Delegate: os.Stdout, Censor: o.secretAgent}
 	stderr := bumper.HideSecretsWriter{Delegate: os.Stderr, Censor: o.secretAgent}
 
-	sourceBranchName := strings.ReplaceAll(strings.ToLower(prTitle), " ", "-")
+	sourceBranchName := strings.ReplaceAll(strings.ToLower(remoteBranch), " ", "-")
```

Comment:
Why isn't this a breaking change?

Reply:
This is a breaking change but UpsertPr is used in only one other place where I am passing in the prTitle as the remoteBranch.